selfdrive/athena/athenad.py

```diff
@@ -29,7 +29,7 @@
 from selfdrive.swaglog import cloudlog
 
 ATHENA_HOST = os.getenv('ATHENA_HOST', 'wss://athena.comma.ai')
-HANDLER_THREADS = os.getenv('HANDLER_THREADS', 4)
+HANDLER_THREADS = int(os.getenv('HANDLER_THREADS', "4"))
```

Comment:
why make the default a string?

Reply:
because getenv returns a string. This would have broken if you actually passed in an env variable.